package.json

```diff
@@ -5,6 +5,15 @@
   "author": "Mark <mark@remarkablemark.org>",
   "main": "index.js",
   "types": "index.d.ts",
+  "module": "index.mjs",
+  "exports": {
```

Comment:
Is `exports` necessary? See [html-react-parser](https://github.com/remarkablemark/html-react-parser/commit/ca0340a25e9c9c56758656b3352267ade7e37a21#diff-d6299b5bb2afc0956126a0adb12fd89829e01a4cc025e7277a092228b96f32e4)

Reply:
Node.js ignores "module" field and only reads from "main" / "exports". that's why we need it there

doc: https://nodejs.org/api/packages.html#packages_exports

Reply:
Gotcha, thanks for sending the doc. Can we add an example or a test to verify that the field is necessary here?